crates/database/src/jit_orders.rs

```diff
@@ -0,0 +1,218 @@
+use {
+    crate::{
+        orders,
+        orders::{BuyTokenDestination, OrderKind, SellTokenSource, SigningScheme},
+        Address,
+        AppId,
+        OrderUid,
+    },
+    sqlx::{
+        types::{
+            chrono::{DateTime, Utc},
+            BigDecimal,
+        },
+        PgConnection,
+    },
+};
+
+const JIT_ORDERS_SELECT: &str = r#"
+o.uid, o.owner, o.creation_timestamp, o.sell_token, o.buy_token, o.sell_amount, o.buy_amount,
+o.valid_to, o.app_data, o.fee_amount, o.kind, o.signature,
+o.receiver, o.signing_scheme, o.sell_token_balance, o.buy_token_balance,
+(SELECT COALESCE(SUM(t.buy_amount), 0) FROM trades t WHERE t.order_uid = o.uid) AS sum_buy,
+(SELECT COALESCE(SUM(t.sell_amount), 0) FROM trades t WHERE t.order_uid = o.uid) AS sum_sell,
+(SELECT COALESCE(SUM(t.fee_amount), 0) FROM trades t WHERE t.order_uid = o.uid) AS sum_fee,
+COALESCE((SELECT SUM(surplus_fee) FROM order_execution oe WHERE oe.order_uid = o.uid), 0) as executed_surplus_fee
+"#;
```

Comment:
Is it expected to use the same statement in different places? Why not move it straight into the `QUERY` below and drop `concatcp!`?